Sources/PublishCLICore/ProjectGenerator.swift

```diff
@@ -154,12 +154,21 @@ private extension Folder {
 }
 
 private extension String {
-    func asSiteName() -> Self {
-        let letters = filter { $0.isLetter }
-        guard !letters.isEmpty else {
+    var asSiteName: Self {
```

Comment:
This should not be converted to a property, since it performs `O(>1)` computation. Refer to [this article](https://www.swiftbysundell.com/articles/computed-properties-in-swift) for more detailed reasoning.